configuration/config-txt/boot.md

```diff
@@ -99,12 +99,37 @@ If `disable_splash` is set to `1`, the rainbow splash screen will not be shown o
 
 On the Raspberry Pi 4B, if this value is set to `0` then the interrupts will be routed to the ARM cores using the legacy interrupt controller, rather than via the GIC-400. The default value is `1`.
 
+## os_prefix
+<a name="os_prefix">
+
+The value in `os_prefix` is prepended to (stuck in front of) the name of any operating system files loaded by the firmware, where "operating system files" is defined to mean kernels, initramfs, cmdline.txt, .dtbs and overlays. The prefix would commonly be a directory name, but it could also be part of the filename such as "test-". For this reason, directory prefixes must include the trailing `/` character.
+
+In an attempt to reduce the chance of a non-bootable system, the firmware first tests the supplied prefix value for viability - unless the expected kernel and .dtb can be found at the new location/name, the prefix is ignored (set to ""). A special case of this viability test is applied to overlays, which will only be loaded from `${os_prefix}${overlay_prefix}` (where the default value of [`overlay_prefix`](#overlay_prefix) is "overlays/") if `${os_prefix}${overlay_prefix}README` exists, otherwise it ignores `os_prefix` and treats overlays as shared.
```

Comment:
Should the "A special case of this viability test is applied to overlays..." sentence be moved to the `overlay_prefix` section rather than being in this `os_prefix` section?

Reply:
No, I don't think so - it is referring to the fallback mechanism that might apply if an os_prefix is set. This is true whatever the overlay_prefix is set to.

Reply:
The latest commit adds the preamble.